.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "TAOS-CI"]
+	path = TAOS-CI
+	url = http://github.com/nnsuite/TAOS-CI.git
```

Comment:
uh..oh...

gitsubmodule is always a troublemaker with OBS/GBS. are you sure you really need this?